recipe/meta.yaml

```diff
@@ -20,9 +19,10 @@ requirements:
   run:
     - python
     - six >=1.2
-    - django >=1.7
-    - typing
-
+    # Pinning Django to versions not failing the testsuite
+    - django ==1.8  # [py3k]
```

Comment:
Can we do `django <=1.8` and avoid the pre-selector?

Reply:
Unfortunately for python 3 the tests only worked for v1.8. I have no time right now, maybe we could put this on hold and discuss what would be the best solution next week?

Reply:
conda will most likely install `django 1.8` in both cases anyway, that is why I'm puzzled by your selector here. Let me try something...

Reply:
> Well this django-extensions version is used in a project we would like to add to conda-forge. However it also pins Django to version 1.7.11 (that's why I could not be too restrictive and used the selector)

`<=1.8` is less restrict than `==1.8` for py3k and we can keep the package `noarch`.

Reply:
Yeah, I meant pinning django==1.8 for which the tests pass independent of the python version.